optimum/exporters/openvino/convert.py

```diff
@@ -109,11 +106,6 @@ def _set_runtime_options(
             or getattr(sub_export_config, "stateful", False)
         ):
             sub_export_config.runtime_options["ACTIVATIONS_SCALE_FACTOR"] = "8.0"
-        if not quantized_model and (
-            "text-generation" in task
-            or ("image-text-to-text" in task and model_name == "language_model")
-            or getattr(sub_export_config, "stateful", False)
-        ):
             sub_export_config.runtime_options["KV_CACHE_PRECISION"] = "f16"
```

Comment:
Not setting "f16" KV cache precision flag for quantized models is not needed because we always remove this flag right after quantization, e.g., [link](https://github.com/huggingface/optimum-intel/blob/25fcb63a9ab9825a41ebaa84852ab18f30a61d31/optimum/intel/openvino/quantization.py#L1673).